### PR TITLE
set nodeports to be disabled by default for prometheus and alertmanager

### PR DESCRIPTION
**Pull request by ceelias**
- [x] Validated by myself

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # SAS Viya Monitoring for Kubernetes
 
+## UNRELEASED
+
+* **Overall**
+
+* **Metrics**
+  * [DEPRECATION] NodePorts is now disabled by default for Prometheus and AlertManager for security reasons. Set the environment variable PROM_NODEPORT_ENABLE=true to replicate previous behavior.
```
Comment (gsmith-sas) [inline CHANGELOG.md:8]: Would prefer describing the change more explicitly; something like: "For security reasons, access to Prometheus and AlertManager via NodePort is no longer enabled by default.  Set the environment variable PROM_NODEPORT_ENABLE=true to replicate previous behavior."

**Review by gsmith-sas — CHANGES_REQUESTED** [review]
Please tweak CHANGELOG wording but otherwise looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (ceelias) [reply CHANGELOG.md]: Makes sense, will do!

**Review by gsmith-sas — APPROVED** [review]
Thanks.  Looks good.

**Pull request merged**